### DISCUSSION TITLE
Rustdesk pricing for Personal Plan

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#4904] [General]
In the description it states "Distributed relay servers" so I take it that means we do not need to run our own servers?

https://rustdesk.com/pricing.html

**Comment by dinger1986** [comment]
It's self hosting pricing on the website

**Reply by ghost** [reply]
Ah okay. Not a problem. 

I'm getting a message

`{"error":"error communicating with stripe: Failed to decode the license type: Not_Ready_Yet_Please_Wait_for_One_more_day"}`

I presume wait till tomorrow to buy?

Will existing clients still be there after upgrading?

**Reply by dinger1986** [reply]
Yeah the payment platform hasn't been completed yet. If you email sales@rustdesk.com I can let you know when it's ready to buy
